cmd/zrok/agentEnroll.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	httptransport "github.com/go-openapi/runtime/client"
+	"github.com/openziti/zrok/agent"
+	"github.com/openziti/zrok/environment"
+	agent2 "github.com/openziti/zrok/rest_client_zrok/agent"
+	"github.com/openziti/zrok/tui"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+func init() {
+	agentCmd.AddCommand(newAgentEnrollCommand().cmd)
+}
+
+type agentEnrollCommand struct {
+	cmd      *cobra.Command
+	headless bool
+}
+
+func newAgentEnrollCommand() *agentEnrollCommand {
+	cmd := &cobra.Command{
+		Use:   "enroll",
+		Short: "Enroll the agent in remote control",
+		Args:  cobra.NoArgs,
+	}
+	command := &agentEnrollCommand{cmd: cmd}
+	cmd.Flags().BoolVar(&command.headless, "headless", false, "Run the agent enrollment in headless mode")
+	cmd.Run = command.run
+	return command
+}
+
+func (cmd *agentEnrollCommand) run(_ *cobra.Command, _ []string) {
+	root, err := environment.LoadRoot()
+	if err != nil {
+		tui.Error("error loading zrokdir", err)
```

Comment:
`zrokdir` one word or a typo? I just don't know if it's supposed to be `zrok dir`

Reply:
it's one word.